lib/ace/ext/language_tools.js

```diff
@@ -224,6 +234,22 @@ require("../config").defineOptions(Editor.prototype, "editor", {
             }
         },
         value: false
+    },
+    completionDelay: {
+        set: function(val) {
+            $completionDelay = val;
```

Comment:
these either need to be added to the editor, or be options on language tools module instead, otherwise changing settings for one editor will change for all other editors too.